<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyTrader v0.4</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>181</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>수동주문</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>계좌</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>15</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>주문</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>45</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>신규매수</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>신규매도</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매수취소</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매도취소</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>종목</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>75</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>135</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>지정가</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>시장가</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>종류</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>수량</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>165</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>가격</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>195</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>10000000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>현금주문</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>105</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>9</y>
      <width>630</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>잔고 및 보유종목현황</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>610</width>
       <height>65</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <row/>
     <column>
      <property name="text">
       <string>예수금 (d+2)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총매입</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총평가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총수익률 (%)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>추정자산</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>544</x>
       <y>410</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>조회</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>-140</y>
       <width>611</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>잔고확인</string>
     </property>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>591</width>
        <height>61</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>예수금</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총매입</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총평가</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총손익</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총수익률</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>추정자산</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>현금주문</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>414</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>실시간 조회</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>610</width>
       <height>311</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>종목명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>보유량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>매입가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>현재가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>평가손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>수익률 (%)</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>630</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>자동 선정 종목 리스트</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>610</width>
       <height>171</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>주문유형</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>종목명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>호가구분</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>수량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>가격</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>상태</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
